src/server/wesnothd/game.cpp

```diff
@@ -74,14 +74,14 @@ namespace wesnothd
 int game::id_num = 1;
 int game::db_id_num = 1;
 
-void game::missing_user(socket_ptr /*socket*/, const std::string& func) const
+void game::missing_user(player_iterator /*socket*/, const std::string& func) const
```

Comment:
Maybe at this point just remove this argument and update the warning text to not mention the socket?

Reply:
Also considering to remove this entire function.